src/sentry/integrations/slack/actions/notification.py

```diff
@@ -38,6 +40,11 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
             "channel_id": {"type": "string", "placeholder": "e.g., CA2FRA079 or UA1J9RTE1"},
             "tags": {"type": "string", "placeholder": "e.g., environment,user,my_tag"},
         }
+        if features.has("organizations:slack-formatting-update", self.project.organization):
+            self.form_fields["mentions"] = {
+                "type": "string",
+                "placeholder": "e.g. @colleen, @on-call-team",
```

Comment:
nice job getting your handle in here lol

Reply:
oh I actually meant to change that 😬